inst/@symfun/minus.m

```diff
@@ -74,14 +74,14 @@
 %! f(x) = x^2;
 %! g = x^2;
 %! s = warning('off', 'OctSymPy:sym:arithmetic:workaround42735');
-%! h = x - f;  assert(isa(h, 'symfun') && isequal(h.sym, x - g))
-%! h = x + f;  assert(isa(h, 'symfun') && isequal(h.sym, x + g))
-%! h = x * f;  assert(isa(h, 'symfun') && isequal(h.sym, x * g))
-%! h = x / f;  assert(isa(h, 'symfun') && isequal(h.sym, x / g))
-%! h = x ^ f;  assert(isa(h, 'symfun') && isequal(h.sym, x ^ g))
-%! h = x .* f; assert(isa(h, 'symfun') && isequal(h.sym, x .* g))
-%! h = x ./ f; assert(isa(h, 'symfun') && isequal(h.sym, x ./ g))
-%! h = x .^ f; assert(isa(h, 'symfun') && isequal(h.sym, x .^ g))
+%! h = f - x;  assert(isa(h, 'symfun') && isequal(h.symbol, g - x))
+%! h = f + x;  assert(isa(h, 'symfun') && isequal(h.symbol, g + x))
+%! h = f * x;  assert(isa(h, 'symfun') && isequal(h.symbol, g * x))
+%! h = f / x;  assert(isa(h, 'symfun') && isequal(h.symbol, g / x))
+%! h = f ^ x;  assert(isa(h, 'symfun') && isequal(h.symbol, g ^ x))
+%! h = f .* x; assert(isa(h, 'symfun') && isequal(h.symbol, g .* x))
+%! h = f ./ x; assert(isa(h, 'symfun') && isequal(h.symbol, g ./ x))
+%! h = f .^ x; assert(isa(h, 'symfun') && isequal(h.symbol, g .^ x))
```

Comment:
>  If it's ok I can add conditions in ...

Dunno, try I guess.  My impression was if we cannot call the subclass method than all the subclass code will have to be duplicated in `@sym`... ick!  Fix upstream would be best of all of course!